docs/pages/keyconcepts/templating/templating.rst

```diff
@@ -34,39 +34,97 @@ Currently, the following data is available through templating:
++---------------------------------------------------------------------+-----------------------------------------------------------+-----------------------------------------+
+| A random email address                                              | {{ randomEmail }}                                         |  LoriStewart@Photolist.com              |
++---------------------------------------------------------------------+-----------------------------------------------------------+-----------------------------------------+
+| A random IPv4  address                                              | {{ randomIPv4 }}                                          |  224.36.27.8                            |
++---------------------------------------------------------------------+-----------------------------------------------------------+-----------------------------------------+
+| A random IPv6  address                                              | {{ randomIPv6 }}                                          |  41d7:daa0:6e97:6fce:411e:681:f86f:e557 |
++---------------------------------------------------------------------+-----------------------------------------------------------+-----------------------------------------+
+| A random UUID                                                       | {{ randomUuid }}                                          |  7b791f3d-d7f4-4635-8ea1-99568d821562   |
++---------------------------------------------------------------------+-----------------------------------------------------------+-----------------------------------------+
+
+Durations
+~~~~~~~~~
+When using template helper methods such as ``currentDateTimeAdd`` and ``currentDateTimeSubtract``, durations must be formatted following the following syntax for durations. 
+
++-----------+-------------+
+| Shorthand | Type        |
++===========+=============+
+| ns        | Nanosecond  |
++-----------+-------------+
+| us/µs     | Microsecond |
++-----------+-------------+
+| ms        | Millisecond |
++-----------+-------------+
+| s         | Second      |
++-----------+-------------+
+| m         | Minute      |
++-----------+-------------+
+| h         | Hour        |
++-----------+-------------+
+| d         | Day         |
++-----------+-------------+
+| y         | Year        |
++-----------+-------------+
+
+Example Durations
+~~~~~~~~~~~~~~~~~
+
++-----------+-------------------+
+| 5m        | 5 minutes         |
++-----------+-------------------+
+| 1h30m     | 1 hour 5 minutes  |
++-----------+-------------------+
+| 1y10d     | Millisecond       |
++-----------+-------------------+
+
+Date time formats
+~~~~~~~~~~~~~~~~~
+When using template helper methods such as ``currentDateTime``, ``currentDateTimeAdd`` and ``currentDateTimeSubtract``, date time formats must follow
+the Golang syntax. More can be found out here https://golang.org/pkg/time/#Parse
+
+Example date time formats
```

Comment:
Probably adding  a note about the golang reference date time would help user who is not familiar with Golang date time formatting